littlechurchlv.com/tour/Config_LITTLECHAPEL.txt
scene.num = 1
s0.type = sphere
s0.name = FINAL3000
s0.transitioneffect = fade
s0.img = Scene/FINAL3000.jpg
s0.panmax = 180
s0.panmin = -180
s0.pan = 0
s0.tiltmax = 45
s0.tiltmin = -90
s0.tilt = 0
s0.fovmax = 91
s0.fovmin = 48
s0.fov = 82
s0.movable = true
s0.speedrate = 20

path.num = 1
path.default = 0
path.slideshow = 0
path.autoplay = true
p0.name = _slideshow_
p0.loop = true
p0.frame.num = 2
p0.f0.s = 0
p0.f0.t = 1
p0.f0.v = 0, 0, 82
p0.f0.d = 1
p0.f1.s = 0
p0.f1.t = 22
p0.f1.v = 0, 0, 82
p0.f1.d = 1

hsbar.bgcolor = 212, 208, 200

vsbar.bgcolor = 212, 208, 200

viewer.z = 8
viewer.bounds = 3, 4, 368, 267
viewer.bgcolor = 250, 226, 151
2dimage.num = 1
2d0.z = 7
2d0.bounds = 4, 275, 368, 21
2d0.mode = original
2d0.img = Skin/bar.gif

button.num = 6
btn0.z = 6
btn0.bounds = 14, 277, 17, 17
btn0.style = static
btn0.text.align = center
btn0.text.font = Times New Roman, plain, 10
btn0.text.color = 0, 0, 0
btn0.hint.text = Turning Left
btn0.hint.bgcolor = 255, 255, 225
btn0.img = Skin/left1.gif
btn0.mouseoverimg = Skin/left2.gif
btn0.mousedownimg = Skin/left3.gif
btn0.action = autopanatrate(-20, 0, 0)
btn1.z = 5
btn1.bounds = 36, 277, 17, 17
btn1.style = static
btn1.text.align = center
btn1.text.font = Times New Roman, plain, 10
btn1.text.color = 0, 0, 0
btn1.hint.text = Turning Right
btn1.hint.bgcolor = 255, 255, 225
btn1.img = Skin/right1.gif
btn1.mouseoverimg = Skin/right2.gif
btn1.mousedownimg = Skin/right3.gif
btn1.action = autopanatrate(20, 0, 0)
btn2.z = 4
btn2.bounds = 58, 277, 17, 17
btn2.style = static
btn2.text.align = center
btn2.text.font = Times New Roman, plain, 10
btn2.text.color = 0, 0, 0
btn2.hint.text = Moving Up
btn2.hint.bgcolor = 255, 255, 225
btn2.img = Skin/up1.gif
btn2.mouseoverimg = Skin/up2.gif
btn2.mousedownimg = Skin/up3.gif
btn2.action = autopanatrate(0, 20, 0)
btn3.z = 3
btn3.bounds = 80, 277, 17, 17
btn3.style = static
btn3.text.align = center
btn3.text.font = Times New Roman, plain, 10
btn3.text.color = 0, 0, 0
btn3.hint.text = Moving Down
btn3.hint.bgcolor = 255, 255, 225
btn3.img = Skin/down1.gif
btn3.mouseoverimg = Skin/down2.gif
btn3.mousedownimg = Skin/down3.gif
btn3.action = autopanatrate(0, -20, 0)
btn4.z = 2
btn4.bounds = 102, 277, 17, 17
btn4.style = static
btn4.text.align = center
btn4.text.font = Times New Roman, plain, 10
btn4.text.color = 0, 0, 0
btn4.hint.text = Zooming In
btn4.hint.bgcolor = 255, 255, 225
btn4.img = Skin/zoomin1.gif
btn4.mouseoverimg = Skin/zoomin2.gif
btn4.mousedownimg = Skin/zoomin3.gif
btn4.action = autopanatrate(0, 0, -20)
btn5.z = 1
btn5.bounds = 124, 277, 17, 17
btn5.style = static
btn5.text.align = center
btn5.text.font = Times New Roman, plain, 10
btn5.text.color = 0, 0, 0
btn5.hint.text = Zooming Out
btn5.hint.bgcolor = 255, 255, 225
btn5.img = Skin/zoomout1.gif
btn5.mouseoverimg = Skin/zoomout2.gif
btn5.mousedownimg = Skin/zoomout3.gif
btn5.action = autopanatrate(0, 0, 20)

iniversion = 1.25


walmart/WMDiscountEmails/WMDiscountExport.exe.config_hold.txt
<?xml version="1.0" encoding="utf-8" ?>
<configuration>
  <appSettings>
    
    <add key="SMTPHost" value="mail.wbsrecords.com"/>
    <add key="SMTPPort" value="25"/>
    <add key="SMTPUser" value="Wal-Mart@wbswireless.com"/>
    <add key="SMTPPass" value="kua28n69"/>
    <add key="SMTPSender" value="Wal-Mart Associate Discounts &lt;Wal-Mart@wbswireless.com&gt;" />
<!--    <add key="SMTPRecipient" value="Brett Jackson &lt;bj6351@att.com&gt;" />-->
    <add key="SMTPRecipient" value="Eric Standard &lt;ES7141@att.com&gt;" />
<!--    <add key="SMTPRecipient" value="&lt;ew8662@att.com&gt;" />-->
    <add key="SMTPCC" value="Rick Stapp &lt;wmreport@wbswireless.com&gt;" />
    <add key="SMTPBCC" value="Jeff Saunders &lt;j.saunders@wbsrecords.com&gt;" />

  </appSettings>
  <connectionStrings>
    <add name="SQLCnx" connectionString="Server=(local);Database=WBSRecords;uid=WBSWeb;pwd=00WBSODBC00;" />
  </connectionStrings>
</configuration>


jandeen.com/web/wp-content/plugins/cbnet-really-simple-captcha-comments/settings.php
<?php
/**
 * Plugin Settings API Implementation
 */

/**
 * Globalize Plugin options
 */
global $cbnet_rscc_options;
$cbnet_rscc_options = cbnet_rscc_get_options();

/**
 * Get Plugin options
 */
function cbnet_rscc_get_options() {
	return wp_parse_args( get_option( 'plugin_cbnet_rscc_options', array() ), cbnet_rscc_get_option_defaults() );
}

/**
 * Get Plugin option defaults
 */
function cbnet_rscc_get_option_defaults() {
	$defaults = array(	
		'chars' => 'ABCDEFGHJKLMNPQRSTUVWXYZ23456789',
		'char_length' => '4',
		//'img_size' => array( '72', '24' ),
		'img_size_x' => '72',
		'img_size_y' => '24',
		'fg_r' => '0',
		'fg_g' => '0',
		'fg_b' => '0',
		//'fg' => array( '0', '0', '0' ),
		'bg_r' => '255',
		'bg_g' => '255',
		'bg_b' => '255',
		//'bg' => array( '255', '255', '255' ),
		'font_size' => '16',
		'font_char_width' => '15',
		'img_type' => 'png',
		'base' => array( '6', '18'),
		'comment_form_label' => __( 'Anti-Spam' , 'cbnet_rscc' )
	);
	return apply_filters( 'cbnet_rscc_option_defaults', $defaults );
}

/**
 * Get Plugin option parameters
 */
function cbnet_rscc_get_option_parameters() {
	$defaults = cbnet_rscc_get_option_defaults();
	$parameters = array(
		'chars' => array(
			'name' => 'chars',
			'title' => __( 'Characters' , 'cbnet_rscc' ),
			'description' => __( 'Characters available to be used in generated CAPTCHA images' , 'cbnet_rscc' ),
			'type' => 'textarea',
			'sanitize' => 'nohtml',
			'default' => $defaults['chars']
		),
		'char_length' => array(
			'name' => 'char_length',
			'title' => __( 'Character Length' , 'cbnet_rscc' ),
			'description' => __( 'Number of characters used in generated CAPTCHA images' , 'cbnet_rscc' ),
			'type' => 'text',
			'sanitize' => 'integer',
			'default' => $defaults['char_length']
		),
		'img_size_x' => array(
			'name' => 'img_size_x',
			'title' => __( 'Image Width' , 'cbnet_rscc' ),
			'description' => __( 'Width of CAPTCHA image, in pixels (px)' , 'cbnet_rscc' ),
			'type' => 'text',
			'sanitize' => 'integer',
			'default' => $defaults['img_size_x']
		),
		'img_size_y' => array(
			'name' => 'img_size_y',
			'title' => __( 'Image Height' , 'cbnet_rscc' ),
			'description' => __( 'Height of CAPTCHA image, in pixels (px)' , 'cbnet_rscc' ),
			'type' => 'text',
			'sanitize' => 'integer',
			'default' => $defaults['img_size_y']
		),
		'fg_r' => array(
			'name' => 'fg_r',
			'title' => __( 'Image Foreground Color R Value' , 'cbnet_rscc' ),
			'description' => __( 'Image foreground RGB color R value' , 'cbnet_rscc' ),
			'type' => 'text',
			'sanitize' => 'rgb',
			'default' => $defaults['fg_r']
		),
		'fg_g' => array(
			'name' => 'fg_g',
			'title' => __( 'Image Foreground Color G Value' , 'cbnet_rscc' ),
			'description' => __( 'Image foreground RGB color R value' , 'cbnet_rscc' ),
			'type' => 'text',
			'sanitize' => 'rgb',
			'default' => $defaults['fg_g']
		),
		'fg_b' => array(
			'name' => 'fg_b',
			'title' => __( 'Image Foreground Color B Value' , 'cbnet_rscc' ),
			'description' => __( 'Image foreground RGB color B value' , 'cbnet_rscc' ),
			'type' => 'text',
			'sanitize' => 'rgb',
			'default' => $defaults['fg_b']
		),
		'bg_r' => array(
			'name' => 'bg_r',
			'title' => __( 'Image Background Color R Value' , 'cbnet_rscc' ),
			'description' => __( 'Image background RGB color R value' , 'cbnet_rscc' ),
			'type' => 'text',
			'sanitize' => 'rgb',
			'default' => $defaults['bg_r']
		),
		'bg_g' => array(
			'name' => 'bg_g',
			'title' => __( 'Image Background Color G Value' , 'cbnet_rscc' ),
			'description' => __( 'Image background RGB color G value' , 'cbnet_rscc' ),
			'type' => 'text',
			'sanitize' => 'rgb',
			'default' => $defaults['bg_g']
		),
		'bg_b' => array(
			'name' => 'bg_b',
			'title' => __( 'Image Background Color B Value' , 'cbnet_rscc' ),
			'description' => __( 'Image background RGB color B value' , 'cbnet_rscc' ),
			'type' => 'text',
			'sanitize' => 'rgb',
			'default' => $defaults['bg_b']
		),
		'font_size' => array(
			'name' => 'font_size',
			'title' => __( 'Font Size' , 'cbnet_rscc' ),
			'description' => __( 'Font size in points (pt) of characters in generated CAPTCHA image' , 'cbnet_rscc' ),
			'type' => 'text',
			'sanitize' => 'integer',
			'default' => $defaults['font_size']
		),
		'font_char_width' => array(
			'name' => 'font_char_width',
			'title' => __( 'Character Width' , 'cbnet_rscc' ),
			'description' => __( 'Width in pixels (px) between characters in generated CAPTCHA image' , 'cbnet_rscc' ),
			'type' => 'text',
			'sanitize' => 'rgb',
			'default' => $defaults['font_char_width']
		),
		'img_type' => array(
			'name' => 'img_type',
			'title' => __( 'Image File Type' , 'cbnet_rscc' ),
			'description' => __( 'File type for generated CAPTCHA image' , 'cbnet_rscc' ),
			'type' => 'select',
			'valid_options' => array( 'png', 'jpeg', 'gif' ),
			'default' => $defaults['img_type']
		),
		'comment_form_label' => array(
			'name' => 'comment_form_label',
			'title' => __( 'Comment Form Label' , 'cbnet_rscc' ),
			'description' => __( 'Text label to display for the CAPTCHA in the comment form' , 'cbnet_rscc' ),
			'type' => 'text',
			'sanitize' => 'nohtml',
			'default' => $defaults['comment_form_label']
		),
		'base' => array(
			'name' => 'base',
			'title' => __( 'CAPTCHA Secret Sauce' , 'cbnet_rscc' ),
			'description' => __( 'If you want to modify this option, please use a filter' , 'cbnet_rscc' ),
			'type' => 'internal',
			'default' => $defaults['base']
		)
	);
	return apply_filters( 'cbnet_rscc_option_parameters', $parameters );
}
 
/**
 * Register Plugin Settings
 */
function cbnet_rscc_register_settings() {

	/**
	* Register Favicon setting
	* 
	* Registers Favicon setting as
	* part of core General settings
	*/
	register_setting( 'discussion', 'plugin_cbnet_rscc_options', 'cbnet_rscc_validate_settings' );

	/**
	 * Add settings section to Settings -> Discussion
	 */
	add_settings_section( 'cbnet_rscc', __( 'cbnet Really Simple CAPTCHA Comment Settings' , 'cbnet_rscc' ), 'cbnet_rscc_settings_section', 'discussion' );	
	
	/**
	 * Discussion settings section callback
	 */
	function cbnet_rscc_settings_section() {
		echo '<p>'. __( 'Configure cbnet Really Simple CAPTCHA Comments settings here.' , 'cbnet_rscc' ) . '</p>';
	}

	/**
	* Add CAPTCHA word setting field
	* 
	* Adds Favicon setting field to 
	* Settings -> General
	*/
	add_settings_field( 'cbnet_rscc_captcha_form', '<label for="cbnet_rscc_captcha_form">' . __( 'Comment Form' , 'cbnet_rscc' ) . '</label>', 'cbnet_rscc_settings_field_captcha_form', 'discussion', 'cbnet_rscc' );
	
	/**
	 * CAPTCHA word setting fields callback
	 */
	function cbnet_rscc_settings_field_captcha_form() {
		global $cbnet_rscc_options;
		$option_parameters = cbnet_rscc_get_option_parameters();
		?>
		<p>
			<input type="text" size="40" name="plugin_cbnet_rscc_options[comment_form_label]" value="<?php echo esc_attr( $cbnet_rscc_options['comment_form_label'] ); ?>" />
			<?php echo $option_parameters['comment_form_label']['description']; ?>
		</p>
		<?php
	}

	/**
	* Add CAPTCHA word setting field
	* 
	* Adds Favicon setting field to 
	* Settings -> General
	*/
	add_settings_field( 'cbnet_rscc_captcha_word', '<label for="cbnet_rscc_captcha_word">' . __( 'CAPTCHA Word' , 'cbnet_rscc' ) . '</label>', 'cbnet_rscc_settings_field_captcha_word', 'discussion', 'cbnet_rscc' );
	
	/**
	 * CAPTCHA word setting fields callback
	 */
	function cbnet_rscc_settings_field_captcha_word() {
		global $cbnet_rscc_options;
		$option_parameters = cbnet_rscc_get_option_parameters();
		?>
		<p>
			<?php echo $option_parameters['chars']['description']; ?><br />
			<textarea name="plugin_cbnet_rscc_options[chars]" cols="80" rows="4"><?php echo esc_textarea( $cbnet_rscc_options['chars'] ); ?></textarea>			 
		</p>
		<p>
			<input type="text" size="4" name="plugin_cbnet_rscc_options[char_length]" value="<?php echo esc_attr( $cbnet_rscc_options['char_length'] ); ?>" />
			<?php echo $option_parameters['char_length']['description']; ?>
		</p>
		<?php
	}

	/**
	* Add CAPTCHA font setting field
	* 
	* Adds Favicon setting field to 
	* Settings -> General
	*/
	add_settings_field( 'cbnet_rscc_captcha_font', '<label for="cbnet_rscc_captcha_font">' . __( 'CAPTCHA Font' , 'cbnet_rscc' ) . '</label>', 'cbnet_rscc_settings_field_captcha_font', 'discussion', 'cbnet_rscc' );
	
	/**
	 * CAPTCHA font setting fields callback
	 */
	function cbnet_rscc_settings_field_captcha_font() {
		global $cbnet_rscc_options;
		$option_parameters = cbnet_rscc_get_option_parameters();
		?>
		<p>
			<input type="text" size="4" name="plugin_cbnet_rscc_options[font_size]" value="<?php echo esc_attr( $cbnet_rscc_options['font_size'] ); ?>" />
			<?php echo $option_parameters['font_size']['description']; ?>
		</p>
		<p>
			<input type="text" size="4" name="plugin_cbnet_rscc_options[font_char_width]" value="<?php echo esc_attr( $cbnet_rscc_options['font_char_width'] ); ?>" />
			<?php echo $option_parameters['font_char_width']['description']; ?>
		</p>
		<?php
	}

	/**
	* Add CAPTCHA image setting field
	* 
	* Adds Favicon setting field to 
	* Settings -> General
	*/
	add_settings_field( 'cbnet_rscc_captcha_image', '<label for="cbnet_rscc_captcha_image">' . __( 'CAPTCHA Image' , 'cbnet_rscc' ) . '</label>', 'cbnet_rscc_settings_field_captcha_image', 'discussion', 'cbnet_rscc' );
	
	/**
	 * CAPTCHA image setting fields callback
	 */
	function cbnet_rscc_settings_field_captcha_image() {
		global $cbnet_rscc_options;
		$option_parameters = cbnet_rscc_get_option_parameters();
		?>
		<p>
			<input type="text" size="4" name="plugin_cbnet_rscc_options[img_size_x]" value="<?php echo esc_attr( $cbnet_rscc_options['img_size_x'] ); ?>" />
			<?php echo $option_parameters['img_size_x']['description']; ?>
			<br />
			<input type="text" size="4" name="plugin_cbnet_rscc_options[img_size_y]" value="<?php echo esc_attr( $cbnet_rscc_options['img_size_y'] ); ?>" />
			<?php echo $option_parameters['img_size_y']['description']; ?>
		</p>
		<p>
			<input type="text" size="4" name="plugin_cbnet_rscc_options[fg_r]" value="<?php echo esc_attr( $cbnet_rscc_options['fg_r'] ); ?>" />
			<?php echo $option_parameters['fg_r']['description']; ?>
			<br />
			<input type="text" size="4" name="plugin_cbnet_rscc_options[fg_g]" value="<?php echo esc_attr( $cbnet_rscc_options['fg_g'] ); ?>" />
			<?php echo $option_parameters['fg_g']['description']; ?>
			<br />
			<input type="text" size="4" name="plugin_cbnet_rscc_options[fg_b]" value="<?php echo esc_attr( $cbnet_rscc_options['fg_b'] ); ?>" />
			<?php echo $option_parameters['fg_b']['description']; ?>
		</p>
		<p>
			<input type="text" size="4" name="plugin_cbnet_rscc_options[bg_r]" value="<?php echo esc_attr( $cbnet_rscc_options['bg_r'] ); ?>" />
			<?php echo $option_parameters['bg_r']['description']; ?>
			<br />
			<input type="text" size="4" name="plugin_cbnet_rscc_options[bg_g]" value="<?php echo esc_attr( $cbnet_rscc_options['bg_g'] ); ?>" />
			<?php echo $option_parameters['bg_g']['description']; ?>
			<br />
			<input type="text" size="4" name="plugin_cbnet_rscc_options[bg_b]" value="<?php echo esc_attr( $cbnet_rscc_options['bg_b'] ); ?>" />
			<?php echo $option_parameters['bg_b']['description']; ?>
		</p>
		<p>
			<?php
			$valid_img_type_options = $option_parameters['img_type']['valid_options'];
			?>
			<select name="plugin_cbnet_rscc_options[img_type]">
			<?php 
			foreach ( $valid_img_type_options as $valid_option ) {
				?>
				<option <?php selected( $valid_option == $cbnet_rscc_options['img_type'] ); ?> value="<?php echo $valid_option; ?>"><?php echo $valid_option; ?></option>
				<?php
			}
			?>
			</select>
		</p>
		<?php
	}
}
add_action( 'admin_init', 'cbnet_rscc_register_settings' );



/**
 * Plugin register_setting() sanitize callback
 * 
 * Validate and whitelist user-input data before updating Plugin 
 * Options in the database. Only whitelisted options are passed
 * back to the database, and user-input data for all whitelisted
 * options are sanitized.
 * 
 * @link	http://codex.wordpress.org/Data_Validation	Codex Reference: Data Validation
 * 
 * @param	array	$input	Raw user-input data submitted via the Plugin Settings page
 * @return	array	$input	Sanitized user-input data passed to the database
 */
function cbnet_rscc_validate_settings( $input ) {

	// This is the "whitelist": current settings
	global $cbnet_rscc_options;
	$valid_input = $cbnet_rscc_options;
	// Get the array of option parameters
	$option_parameters = cbnet_rscc_get_option_parameters();
	// Get the array of option defaults
	$option_defaults = cbnet_rscc_get_option_defaults();
	
	// Determine what type of submit was input
	$submittype = ( ! empty( $input['reset'] ) ? 'reset' : 'submit' );	
	
	// Loop through each setting
	foreach ( $option_defaults as $setting => $value ) {
		
		// If submit, validate/sanitize $input
		if ( 'submit' == $submittype ) {
		
			// Get the setting details from the defaults array
			$optiondetails = $option_parameters[$setting];
			// Get the array of valid options, if applicable
			$valid_options = ( isset( $optiondetails['valid_options'] ) ? $optiondetails['valid_options'] : false );
			
			// Validate checkbox fields
			if ( 'checkbox' == $optiondetails['type'] ) {
				// If input value is set and is true, return true; otherwise return false
				$valid_input[$setting] = ( ( isset( $input[$setting] ) && true == $input[$setting] ) ? true : false );
			}
			// Validate radio button fields
			else if ( 'radio' == $optiondetails['type'] ) {
				// Only update setting if input value is in the list of valid options
				$valid_input[$setting] = ( array_key_exists( $input[$setting], $valid_options ) ? $input[$setting] : $valid_input[$setting] );
			}
			// Validate select fields
			else if ( 'select' == $optiondetails['type'] ) {
				// Only update setting if input value is in the list of valid options
				$valid_input[$setting] = ( in_array( $input[$setting], $valid_options ) ? $input[$setting] : $valid_input[$setting] );
			}
			// Validate text input and textarea fields
			else if ( ( 'text' == $optiondetails['type'] || 'textarea' == $optiondetails['type'] ) ) {
				// Validate no-HTML content
				if ( 'nohtml' == $optiondetails['sanitize'] ) {
					// Pass input data through the wp_filter_nohtml_kses filter
					$valid_input[$setting] = wp_filter_nohtml_kses( $input[$setting] );
				}
				// Validate HTML content
				if ( 'html' == $optiondetails['sanitize'] ) {
					// Pass input data through the wp_filter_kses filter
					$valid_input[$setting] = wp_filter_kses( $input[$setting] );
				}
				// Validate integer content
				if ( 'integer' == $optiondetails['sanitize'] ) { 
					// Verify value is an integer
					$valid_input[$setting] = ( is_int( (int) $input[$setting] ) ? $input[$setting] : $valid_input[$setting] );
				}
				// Validate RGB content
				if ( 'rgb' == $optiondetails['sanitize'] ) { 
					// Verify value is an integer
					$valid_input[$setting] = ( ( is_int( (int) $input[$setting] ) && 0 <= (int) $input[$setting] && 255 >= (int) $input[$setting] ) ? $input[$setting] : $valid_input[$setting] );
				}
			}
		} 
		// If reset, reset defaults
		elseif ( 'reset' == $submittype ) {
			// Set $setting to the default value
			$valid_input[$setting] = $option_defaults[$setting];
		}
	}
	return $valid_input;		

}
?>